enhancements/two-node-fencing/tnf.md

```diff
@@ -287,28 +287,34 @@ scope of this enhancement proposal and should be detailed in its own enhancement
 #### etcd Operator Changes
 
 Initially, the creation of an etcd cluster will be driven in the same way as other platforms. Once the cluster has two members, the etcd daemon will be removed from the static pod definition and
-recreated as a resource controlled by RHEL-HA. At this point, the cluster-etcd-operator (CEO) will be made aware of this change so that some membership management functionality that is now handled by
-RHEL-HA can be disabled. This will be achieved by having the same entity that drives the configuration of RHEL-HA use the OpenShift API to update a field in the CEO's `ConfigMap` - which can only
-succeed if the control-plane is healthy.
+recreated as a resource controlled by RHEL- a resource agent called podman-etcd. At this point, the cluster-etcd-operator (CEO) will be made aware of this change so that some membership management
```

Comment:
is this wanted, or the "RHEL-" part was also intended to be removed?

Reply:
I think something happened by accident. RHEL-HA was meant to stay :)